src/index.html

```diff
@@ -0,0 +1,127 @@
+<html lang="pl">
+<head>
+  <meta charset="UTF-8">
+  <title>Github-browser</title>
+</head>
+<body>
+
+<nav class="navbar">
+  <div class="navbar-brand">
+    <div class="navbar-item">
+      <div class="field has-addons">
+        <div class="control">
+          <input class="input username" type="text" placeholder="enter github username">
```

Comment:
For this control add `id`, `name` attributes and also add `<label></label>` for this input to increase accesibility. 
Put everything in `<form></form>` with id